src/common/systems/sysuniversegenerator.cpp

```diff
@@ -71,4 +85,25 @@ void cqsp::common::systems::universegenerator::ScriptUniverseGenerator::Generate
             (*generator)["planets"](ent);
         }
     }
+
+
+
+   // add first ship(could be deferred to some script)
+   //has to be deferred until after the galaxy and systems are populated in the scripts
+    
+    auto starting_planet =
+        universe.get<cqspc::Civilization>(player).starting_planet;
+    
+    
+    cqsp::common::systems::actions::CreateShip(
+        universe, playerFleet,
+        universe.get<cqspc::bodies::Body>(starting_planet).star_system,
+        starting_planet,
+        "pioneer");
+      cqsp::common::systems::actions::CreateShip(
+        universe, playerSubFleet,
+        universe.get<cqspc::bodies::Body>(starting_planet).star_system,
+        starting_planet,
+        "pioneer2");
```

Comment:
Ships can be created by scripts, make this a script.

Reply:
its a bit hard to do that, I have it planned though